app/templates/content/framework-crown-hosting.html

```diff
@@ -0,0 +1,276 @@
+      <li><a href="#the-crown-hosting-data-centres-framework">The Crown Hosting Data Centres framework</a></li>
+      <li><a href="#further-information">Further information</a></li>
+    </ol></nav>
+    </div>
+    </header>
+    <div class="block publication-content">
+    <div class="inner-block floated-children">
+    <div class="govspeak">
+
+    <h2 id="what-are-frameworks">
+    <span class="number">1. </span>What are frameworks?</h2>
+    <p>
+      Government frameworks are agreements between the government and suppliers. The basic terms of use are agreed by both parties following a formal <a href="http://www.ojeu.eu/" rel="external">Official Journal of the European Union (OJEU)</a> procurement process. This saves public sector organisations and suppliers the time and cost traditionally associated with individual procurement contracts. Buyers and suppliers still need to sign a ‘call-off contract’ for each service procured through a framework.
+    </p>
+
+
+    <h2 id="frameworks-on-the-digital-marketplace">
+    <span class="number">2. </span>Frameworks on the Digital Marketplace</h2>
+    <p>
+      Public sector organisations can use the Digital Marketplace to access services from 3 frameworks:
+    </p>
+    <ul>
+      <li>
+        the G-Cloud framework, which includes cloud technology and support (eg web hosting or IT health checks)
+      </li>
+      <li>
+        the Digital Services framework, which includes specialists for work on specific digital projects (eg technical architects and user researchers)
+      </li>
+      <li>
+        the Crown Hosting Data Centres framework, which provides access to physical datacentre space for legacy systems
+      </li>
+    </ul>
+
+
+
+    <h2 id="the-crown-hosting-data-centres-framework">
+    <span class="number">3. </span>The Crown Hosting Data Centres framework</h2>
+    <p>
+      The Crown Hosting Data Centres framework is an agreement between the government and a single supplier called <a href="http://crownhostingdc.co.uk/" rel="external">Crown Hosting Data Centres</a>, which is a joint venture between the government and Ark Data Centres. The framework provides public sector organisations access to physical datacentre space for legacy systems.
+    </p>
+    <p>
+      The Crown Hosting Data Centres framework supplements the government’s <a href="https://www.gov.uk/government/news/government-adopts-cloud-first-policy-for-public-sector-it">Cloud First</a> policy. Although Cloud First has helped the government move many applications to the cloud, some legacy applications either can’t make the transition in the short term or are too expensive to switch. This pan-government framework ensures that these applications can continue to be hosted for as long as is necessary, cheaply and securely.
+    </p>
+    <h3>
+      Features
+    </h3>
+    <p>
+      The Crown Hosting service offers:
```

Comment:
The spacing here under 'Features' and 'Benefits' doesn't look right (and is different from the existing site):
![screen shot 2015-05-13 at 10 35 55](https://cloud.githubusercontent.com/assets/6525554/7607752/e039c4c2-f95b-11e4-8dbd-4de44676a830.png)